Rewrite all instances of "osu!standard" as "osu!" in the Article Styling Criteria (#5246)
ppy/osu-wiki: wiki/Article_styling_criteria/Writing/en.md

Gameplay elements must not be capitalised, unless they act as a title for a link that points to the article. The following is an example:

```markdown

> In osu!, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.

Reviewer: ```suggestion
In osu! mode, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
```

Reviewer 2: Now needs definite article because English is weird
"In the osu! mode"

Reviewer: Yes, it does look better.

Author: decided to go with `the osu! game mode` as most pages (the Ranking Criteria, the World Cup articles, etc) are writing it this way instead

![image](https://user-images.githubusercontent.com/36564236/114734980-f2c6a480-9d6e-11eb-9d4e-cf50623a0c40.png)

Reviewer: That's what I did here: #5255 

However, in some cases this is not applicable, so I left `osu! mode`.

Game modes must be written as follows:

- `osu!`
- `osu!taiko`
- `osu!catch`
- `osu!mania`

> Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note however that citing the osu! game mode as `osu!standard` is *not* recommended.

Reviewer 3: "reccomended" doesn't really seem to be the best word to describe "not standard". Here's a few options:
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that `osu!standard` should *not* be used to refer to the osu! game mode unless speaking about it's colloquial nature.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that citing the osu! game mode as `osu!standard` is *not* prohibited unless it is speaking of its colloquial nature.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that `osu!standard` should *not* be used to refer to the osu! game mode.
```
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note, however, that citing the osu! game mode as `osu!standard` is *not* prohibited.
```

Reviewer 4: Why are there four suggestions here?

Reviewer 3: I wasn't sure if the exception to the rule should be specified, so it's just different ways of saying the same thing. Only one will be applied tho, so ¯\\\_(ツ)_/¯

Reviewer 2: All of these seem either too verbose or flip the statement around with "is not recommended" -> "is not prohibited", which to me doesn't make much sense. The previous sentence should make that clear enough.

How about
```suggestion
Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do not otherwise cite the osu! game mode as `osu!standard`.
```
(yes, i'm adding another suggestion)

Reviewer 3: The previous sentence does not refer to osu!standard at all, so why "otherwise"?. Doesn't really matter, though, cus this works fine, too. As long as the rule surrounding the usage of "osu!standard" is understandable and made clear, then it should be fine; just needed to get rid of "recommended".

Reviewer 2: Hmm, right, and although a direct "do not cite" may be correct, it doesn't flow well in the paragraph. I'd go with your third suggestion in that case.

Author: o thanks for all the suggestions guys but i guess i'll just go with TicClick's solution in the other comment chain to delete this problematic sentence altogether :x

> Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note however that citing the osu! game mode as `osu!standard` is *not* recommended — unless necesary, `osu!` should be used instead.

Reviewer 5: please define "unless necessary" if you use this sentence. personally I think we may be able to rewrite the wiki in a way that it would always be clear which of the two entities (the game, or the main mode) is mentioned, so no such sentence is needed

Author: ah, okok, removed

Reviewer: > please define "unless necessary" if you use this sentence. personally I think we may be able to rewrite the wiki in a way that it would always be clear which of the two entities (the game, or the main mode) is mentioned, so no such sentence is needed

On this point I was thinking of making a disambiguation for the term `osu!`, or if existing include this new point.

Because I'm thinking of renaming `osu!standard` > `osu!`, so I'm also thinking of a solution to not confuse the two.

Reviewer 5: nah, what I meant is that you didn't even need to mention "osu!standard" as an alternative spelling if it will be removed (and I guess it will)

Reviewer 4: You can call it "osu! (ruleset)" or "osu! mode" or anything. But don't call it osu!standard. It's not how I want it referenced and it sounds bad.

Author: @TicClick ah i see, i guess i was thinking too much when i wrote that sentence down then ;~;

removed the mention of the alternative spelling, should be good now!

Game modes must be written as follows:

> - `osu!`

Reviewer: ```suggestion
- `osu!` (add `mode` when will be needed)
```

Reviewer 2: ``add `mode` when needed``

Author: hmm i genuinely don't think that an instruction \*this\* specific is needed tho. i think needless to say all writers by default are already expected to simply apply common sense on a case-by-case basis here whether to write `osu!`, `osu! mode`, `osu! game mode`, or any variations of it in a sentence :(

Reviewer: I have already found variates like you describe in the articles, each one has its own way of using the term `osu!` to describe the main game mode, to avoid confusion with the game name.

In the end I would say don't worry too much about it, and formulate it the way you think it should be so that it can be understood.

Without using the term `standard`, because indeed I found it quite misplaced. Indeed we have more chances to confuse the two, but if we talk about spinners or hit objects, we can't go wrong I think.

Honestly, it's really case by case how to name the game mode, we can't define a universal term, because it won't fit well in some cases.

In our case, I think we can leave it as it is, and then I think (with practice), we will know how to use it, maybe it's just a matter of adaptation.
